build/README.md

```diff
@@ -1,60 +1,170 @@
+# Start with docker compose
+
+You can start ZITADEL with a simple docker compose up.
+
+The services are configured to restart if an error occurs. Until the essential services are started and executed some services panic, this is expected just give it some minutes to setup the database and execute migrations.
+
+```bash
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose-dev.yml --profile init --profile backend --profile frontend -p zitadel up
+```
 
 # Development
```

Comment:
There are multiple `H1` we should evade this.
You can use a MD linter in VSCode to highlight this.